explorer/src/pages/Attestations/table/columns.tsx

```diff
@@ -0,0 +1,69 @@
+import { ColumnDef } from "@tanstack/react-table";
```

Comment:
move this file to `linea-attestation-registry/explorer/src/constants/columns/myAttestation.tsx`